sorts/exchange_sort.py

```diff
@@ -0,0 +1,25 @@
+def exchange_sort(numbers: list[int]) -> list[int]:
+    """
+    Uses exchange sort to sort a list of numbers.
+    Source: https://en.wikipedia.org/wiki/Sorting_algorithm#Exchange_sort
+    >>> exchange_sort([5, 4, 3, 2, 1])
+    [1, 2, 3, 4, 5]
+    >>> exchange_sort([-1, -2, -3])
+    [-3, -2, -1]
+    >>> exchange_sort([1, 2, 3, 4, 5])
+    [1, 2, 3, 4, 5]
+    >>> exchange_sort([0, 10, -2, 5, 3])
+    [-2, 0, 3, 5, 10]
+    >>> exchange_sort([])
+    []
+    """
+    for i in range(len(numbers)):
+        for j in range(i + 1, len(numbers)):
```

Comment:
Calling `len(numbers)` each iteration is inefficient. I would prefer create local variable like `numbers_count = len(numbers)` and use it then in for loops.